### PR TITLE
Baobab: Update to 3.22.1

### PR DESCRIPTION
**Pull request by Sir-Boops**

### CODE REVIEW

File: srcpkgs/baobab/template [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Template file for 'baobab'
 pkgname=baobab
-version=3.22.0
+version=3.22.1
 revision=1
```
Comment (SirDoctors) [inline srcpkgs/baobab/template]: Please bump the revision number

Comment (ebfe) [inline srcpkgs/baobab/template]: why?

Comment (SirDoctors) [inline srcpkgs/baobab/template]: I was unaware of the standard practice for revision numbers.
@Sir-Boops sorry about that, please revert back to not changing the revision number.

**Comment by Sir-Boops** [comment]
Closing as this was updated via c48359414fa9b0f11e742cfd98c804c4f66e24e3

**Pull request closed without merging**